content/blog/how-to-use-dsc-logging.en.md

```diff
@@ -0,0 +1,120 @@
+---
+title: "How to Use Dsc Logging"
+date: 2019-09-24T16:37:42+02:00
+type: "post"
+draft: false
+author: ykuijs
+---
+
+The Local Configuration Manager (LCM) in PowerShell Desired State Configuration (DSC)
+logs its deployment progress and errors on several places. The most commonly known
+location is the event log, where errors are logged in the log:
+**Applications and Services Logs \ Microsoft \ Windows \ Desired State Configuration \ Operational**
+
+<img src="../../images/dsc_eventlog.png" alt="DSC Eventlog" style="width:725px;" />
+
+This log contains various events that were generated during a DSC job, it is however
+less useful to track the progress of a running job.
+
+### Deployment process
+
+When doing a push deployment using Start-DscConfiguration, you can use the
+-Verbose parameter to output verbose logging to the screen. A downside of this
+verbose logging is that reboots will kill this logging. After the restart DSC
+continues deploying the configuration, without you being able to track its progress.
+The same is true for a Pull deployment, where verbose logging isn't available at all.
+
+Fortunately there is a log that is not known by everybody. During each DSC job
+(even when -Verbose hasn't been specified), all Verbose output is always written
+to log files in: **C:\Windows\System32\Configuration\ConfigurationStatus**
+
+For each job, DSC creates two files:
+
+1. **{\<DSC_Job_Id\>}-0.mof**
+   * This file contains LCM configuration, resources not in the desired state and
+     the results of the DSC job.
+1. **{\<DSC_Job_Id\>}-0.details.json**
+   * This file contains the verbose logging in JSON format.
```

Comment:
I think the numbering went wrong in your last edit

Reply:
@ykuijs You may not have seen this comment ☝️ 
Otherwise it looks good.

Reply:
Numbering in Markdown is always 1. Markdown itself will increment where required.

The built in Markdown validator in VSCode pointed out that the original numbering was incorrect. That is why I corrected it. It should number just fine on the site.

Reply:
It's usually an indentation problem when it's flagged.
No Biggie, no need to hold for that.
I'll merge if your happy with it.
Ok to release?